src/AutoVizuA11y.tsx

```diff
@@ -120,6 +131,8 @@ const AutoVizuA11y = ({
 
 	const dataString = useMemo(() => JSON.stringify(data), data);
 
+	const [visibleShortcutGuide, setVisibleShortcutGuide] = useState(false);
```

Comment:
small np:

prefixig boolean variables such as this with `is`, `isVisibleShortcutGuide`, makes it a lot easier to read and immideatly understand when its used in the rest of the code

Reply:
Given that the ShortcutGuide is now a dialog, this logic changed.